raiden_contracts/contracts/TokenNetworkRegistry.sol

```diff
@@ -16,11 +16,20 @@ contract TokenNetworkRegistry is Utils {
     uint256 public settlement_timeout_min;
     uint256 public settlement_timeout_max;
 
+    // Only for the limited Bug Bounty release
+    address public deprecation_executor;
+    bool public bug_bounty_token_network_created = false;
```

Comment:
please change to `token_network_created` or if you want to include the release name use `red_eyes`.